docs/how-tos/adding-event-bus-support-to-an-event.rst

```diff
@@ -0,0 +1,213 @@
+Adding Event Bus Support to an Open edX Event
+=============================================
+
+Before sending an event across services, you need to ensure that the event is compatible with the Open edX Event Bus. This involves ensuring that the event, with its corresponding payload, can be emitted by a service through the event bus and that it can be consumed by other services. This guide will walk you through the process of adding event bus support to an Open edX event.
+
+For more details on how the :term:`Event Payload` is structured refer to the :doc:`../decisions/0003-events-payload` decision record.
+
+.. note::
+    This guide assumes that you have already created an Open edX event. If you haven't, refer to the :doc:`../how-tos/creating-new-events` how-to guide.
+
+Step 1: Does my Event Need Event Bus Support?
+----------------------------------------------
+
+By default, Open edX Events should be compatible with the Open edX Event Bus. However, there are cases when the support might not be possible or needed for a particular event. Here are some scenarios where you might not need to add event bus support:
+
+- The event is only used within the same application process and cannot be scoped to other services.
+- The :term:`Event Payload` contains data types that are not supported by the event bus, and it is not possible to refactor the :term:`Event Payload` to use supported data types.
+
+When adding support is not possible do the following:
+
+- Add it to the ``KNOWN_UNSERIALIZABLE_SIGNALS`` list in the ``openedx_events/tooling.py`` file so the event bus ignores it.
+- Add a ``warning`` in the event's docstring to inform developers that the event is not compatible with the event bus.
```

Comment:
It would be good to document why as well

Reply:
Agreed! Thank you. https://github.com/openedx/openedx-events/pull/428/commits/d8deec5bec068e884e5f2bdd72d971c687452b76